tests/WE2E/test_configs/verification/config.MET_verification_only_vx.yaml

```diff
@@ -0,0 +1,51 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    can successfully run the MET deterministic verification tasks but
+    using staged observation and forecast files (i.e. with other tasks
+    such as pre-processing, forecast, and post-processing deactivated).
+user:
+  RUN_ENVIR: community
+workflow:
+  PREDEF_GRID_NAME: RRFS_CONUS_25km
+  DATE_FIRST_CYCL: '2019061500'
+  DATE_LAST_CYCL: '2019061500'
+  FCST_LEN_HRS: 6
+  PREEXISTING_DIR_METHOD: rename
+workflow_switches:
+#
+# This test assumes that the post-processed forecast files are staged
+# (i.e., not generated by running the forecast model).  Thus, turn off
+# pre-processing, forecast, post-processing, and other tasks ordinarily 
+# needed for generation of post-processed forecast files.
+#
+  RUN_TASK_MAKE_GRID: false
+  RUN_TASK_MAKE_OROG: false
+  RUN_TASK_MAKE_SFC_CLIMO: false
+  RUN_TASK_GET_EXTRN_ICS: false
+  RUN_TASK_GET_EXTRN_LBCS: false
+  RUN_TASK_MAKE_ICS: false
+  RUN_TASK_MAKE_LBCS: false
+  RUN_TASK_RUN_FCST: false
+  RUN_TASK_RUN_POST: false
+#
+# This test assumes the observation files are staged.  Thus, deactivate
+# the GET_OBS_... tasks and instead specify the obs staging directories.
+#
+  RUN_TASK_GET_OBS_CCPA: false
+  CCPA_OBS_DIR: '/scratch2/BMC/det/Gerard.Ketefian/UFS_CAM/DTC_ensemble_task/staged/obs/ccpa/proc'
```

Comment:
Are there plans to make the staged obs available on other machines?  Or is it planned that this test will only be able to be run on Hera?

Reply:
Thank you very much for the explanation, @gsketefian.  Shall I add this as a topic for an upcoming SRW App CM meeting, so that AUS can assist with getting this data to the NOAAcloud and other machines?

Reply:
@MichaelLueken Yes, good idea!  I don't know what their procedure is, but if it's a significant effort, it might be worthwhile to first get all the data on Hera (including the ensemble vx data) and then do the transfer to other platforms in one go.  That would mean I first have to get a PR into develop for ensemble vx.  We can discuss on Thursday.

Reply:
Seems like it would be easy enough to include obs data in the static data that's already staged. There are already observations staged under `UFS_SRW_App/develop/obs_data/` on all platforms.

Reply:
@mkavulich Yeah as you say there's already a place for obs in the data directories.  We need to decide on where to put staged forecast data from the SRW itself (not from other external models), including ensemble output (which will be needed for a future PR).

Reply:
I am curious why staged forecast data would be needed. Why can't we have a test that runs a forecast and then runs VX on that forecast?

Reply:
@MichaelLueken Sorry just saw this.  The reason is that someone else may have run the forecasts beforehand, and the user needs to only run vx on that output.  For example, for the DTC Ensemble Design task that I'm part of, the RRFS development group at GSL already ran a bunch of ensemble forecasts, and we want to only run vx on it.  In addition, one may want to run vx on forecasts not generated by the SRW, e.g. there's another (I think non-DTC) task that is using the version of the SRW App I've put together in my fork (that I'm now bringing into the develop branch) to run vx on HREF output.